LanguageFeatures/Wildcards/super_parameters_A01_t01.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion An occurrence of `super._` as a declaration of a formal parameter
+/// in a constructor is a compile-time error. This error also occurs in the case
+/// where the super parameter has an explicitly declared type and/or default
+/// value.
+///
+/// @description Checks that it is a compile-time error to refer `super._` as a
+/// formal parameter of a constructor.
```

Comment:
Oh, this was exactly the thing which was changed: It is _no longer_ an error to use `super._`.

The meaning of `super._` is that
- This is a wildcarded parameter declaration, and `_` in an initializer list entry will resolve to an instance/static/top-level declaration, not to the parameter.
- Nevertheless, the compiler is able to generate code for the superinitialization such that the actual argument is passed on to the constructor of the superclass which is called in that manner.

```dart
class A {
  final int i;
  A(this.i);
}

class B extends A {
  B(super._); // Means something like this:
  B(int _freshName2398): super(_freshName2398);
  B.name(super._): assert(_ > 0); // Error, unknown identifier `_`.
}
```